src/main/java/org/ow2/proactive/catalogclient/util/ResponseUtils.java

```diff
@@ -23,21 +23,18 @@
  * If needed, contact us to obtain a release under GPL Version 2 or 3
  * or a different license than the AGPL.
  */
-package org.ow2.proactive.catalogclient.service;
+package org.ow2.proactive.catalogclient.util;
 
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.ow2.proactive.catalogclient.exception.FailedRequestException;
 
+import io.restassured.response.Response;
 
-@ComponentScan(basePackages = { "org.ow2.proactive.catalogclient.*" })
-public class SpringConfig {
 
-    private SpringConfig() {
-    }
+public class ResponseUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1118)